suite-native/storage/src/storage.ts

```diff
@@ -39,33 +53,37 @@ export const clearStorage = () => {
     RNRestart.restart();
 };
 
-// Ideally it should never happen but we need to be sure that at least some message is displayed.
-// If someone will mess with encryptionKey it can corrupt storage and app will crash on startup.
-// Then app will hang on splashscreen indefinitely so we at least want to show some error message.
+const alertUser = () => {
+    SplashScreen.hideAsync();
+    Alert.alert(
+        'We couldn’t load your data',
+        'Try restarting the app first. If the problem persists, you’ll need to wipe this app’s storage.',
+        [
+            {
+                text: 'Reset Storage',
+                onPress: clearStorage,
```

Comment:
We should probably track in Sentry if user decides to click this as error, because it is :D

Reply:
Good idea, added here [`b181a9d` (#16819)](https://github.com/trezor/trezor-suite/pull/16819/commits/b181a9d16fd7fd729fe1aa88186699a5dab110c5)

Example report https://satoshilabs.sentry.io/issues/6272721810/events/08b444862bb1487ba01a70a5cdb1d37a/